src/ui/public/state_management/state.js

```diff
@@ -1,59 +1,94 @@
 import _ from 'lodash';
+import angular from 'angular';
 import rison from 'rison-node';
 import applyDiff from 'ui/utils/diff_object';
 import qs from 'ui/utils/query_string';
 import EventsProvider from 'ui/events';
 import Notifier from 'ui/notify/notifier';
 import KbnUrlProvider from 'ui/url';
 
-const notify = new Notifier();
-export default function StateProvider(Private, $rootScope, $location) {
+import {
+  createStateHash,
+  hashedItemStoreSingleton,
+  isStateHash,
+} from './state_storage';
+
```

Comment:
The issue with putting this on one line is it's then over 80 characters. Also, breaking it up across lines helps with diffs if the future.